recipes/diligent-tools/all/conanfile.py

```diff
@@ -99,6 +119,13 @@ def _configure_cmake(self):
         self._cmake.definitions["DILIGENT_BUILD_TESTS"] = False
         self._cmake.definitions["DILIGENT_BUILD_TOOLS_TESTS"] = False
         self._cmake.definitions["DILIGENT_BUILD_TOOLS_INCLUDE_TEST"] = False
+        self._cmake.definitions["DILIGENT_NO_RENDER_STATE_PACKAGER"] = not self.options.with_render_state_packager
+        if 'cci' not in str(self.version) and (('api' in str(self.version) and Version(self.version) >= Version('api.252005')) or Version(self.version) > Version('2.5.2')):
```

Comment:
Why `Version(self.version) > Version('2.5.2')`? Is api. branch the same as that one using semver.

Reply:
Not sure, what you mean here. 
The following codeblock should be executed for versions higher than 2.5.2
api.25200X is higher

Reply:
the release are all over the place https://github.com/DiligentGraphics/DiligentTools/tags

it's hard to say